client/src/main.rs

```diff
@@ -1034,31 +1040,62 @@ fn main() {
                 .description("Claim the rewards for all meetup participants of the last ceremony.")
                 .options(|app| {
                     app.setting(AppSettings::ColoredHelp)
-                        .signer_arg("account that was part of the meetup")
+                        .signer_arg("Account which signs the tx.")
+                        .meetup_index_arg()
+                        .all_upto_arg()
                 })
                 .runner(move |_args: &str, matches: &ArgMatches<'_>| {
 
                     extract_and_execute(
                         &matches, |api, cid| {
                             let signer = match matches.signer_arg() {
                                 Some(sig) => get_pair_from_str(sig),
-                                None => panic!("please specify --signer. must be an assignee of a recent meetup")
+                                None => panic!("please specify --signer.")
                             };
                             let mut api = api.set_signer(signer.clone().into());
 
                             let tx_payment_cid_arg = matches.tx_payment_cid_arg();
+                            let meetup_index_arg = matches.meetup_index_arg();
+                            let all_upto_arg = matches.all_upto_arg();
                             api = set_api_extrisic_params_builder(api, tx_payment_cid_arg);
-                            let xt: EncointerXt<_> = compose_extrinsic!(
-                                api.clone(),
-                                ENCOINTER_CEREMONIES,
-                                "claim_rewards",
-                                cid,
-                                Option::<MeetupIndexType>::None
-                            );
-                            ensure_payment(&api, &xt.hex_encode(), tx_payment_cid_arg);
 
-                            let _ = api.send_extrinsic(xt.hex_encode(), XtStatus::Ready).unwrap();
-                            println!("Claiming reward for {}. xt-status: 'ready'", signer.public());
+                            if let Some(all_upto) = all_upto_arg {
```

Comment:
now you do nothing if neither meetup-index, nor all_upto were specified. Taht is not what I wanted. Please leave the old behaviour if no extra args are specified

Reply:
well this is what i am doing, setting `meetup_index` to `None` which leads to the old behaviour